$OpenBSD: patch-script_harch_inst,v 1.1 2010/07/27 18:16:19 kili Exp $

No harch black magic for OpenBSD, we don't need this.

--- script/harch.inst.orig	Thu Nov 22 11:32:51 2007
+++ script/harch.inst	Fri Nov 23 21:21:36 2007
@@ -4,6 +4,9 @@
 #		May 1999: ensure that small differences (e.g. kernel version)
 #			  don't matter.
 
+echo OpenBSD
+exit
+
 PROCESSOR=
 OS=
 REL=
